yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1115,3 +1117,29 @@ func validateReportOutputFormat() {
 	}
 	utils.ErrExit("Error: Invalid output format: %s. Supported formats are %v", outputFormat, allowedOutputFormats)
 }
+
+func schemaIsAnalyzed() bool {
```

Comment:
yeah that can also be done but in case of end migration, suppose the schema report is backed up and it fails at some next step. Then this function will report incorrect information.
Just thought of storing info in migrationStatus similar to what we do with `ExportSchemaDone` `ExportDataDone`

Reply:
How about using "copy" while "backing up" the schema report? Then no matter how many times the command runs, it will still work.

Reply:
Or simply check if a schema analysis report is already backed up in the backup-dir. Skip the entire code-path if the report is already present in the backup dir.

Reply:
done